app/models/storage_location.rb

```diff
@@ -211,10 +217,18 @@ def self.csv_export_headers
   end
 
   def csv_export_attributes
-    [name, address, square_footage, warehouse_type, size]
+    attributes = [name, address, square_footage, warehouse_type, total_active_inventory_count]
+    get_active_items.sort { |a, b| a.item.name <=> b.item.name }.each { |item| attributes << item.quantity }
```

Comment:
```suggestion
    get_active_items.sort_by(&:name).each { |item| attributes << item.quantity }
```

Reply:
I need to get the name off the item associated with that inventory item, but I'm not sure how do do that with the shorthand notation, so I changed it to `active_inventory_items.sort_by{ |inv_item| inv_item.item.name }`